src/tools/browser-tools.ts

```
@@ -0,0 +1,254 @@
+  }
+}
+
+/**
+ * Page navigation - scroll down tool
+ */
+export class PageDownTool extends BaseTool {
+  name = 'page_down';
+  description = 'Scroll the viewport DOWN one page-length in the current webpage and return the new viewport content.';
+  schema = z.object({});
+
+  async execute(args: {}, abortSignal?: AbortSignal): Promise<string> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)